src/app/Login.js

```diff
@@ -9,6 +9,7 @@ const Login = withRouter(({history, location}) => {
     const referer = location?.state?.referer || '/problems';
 
     useEffect(() => {
+        document.body.style.backgroundColor = "var(--backgroundDim)"
```

Comment:
Add a TODO comment to explain the reason you have to hardcode the style here and notice the future improvement of the hard-coded code.